src/scriptmanager.h

```diff
@@ -35,7 +35,29 @@ class ScriptingManager
 			return instance;
 		}
 
+		enum ReloadInterface_t {
+			NONE,
```

Comment:
Used in luascript.cpp to avoid having to write something like:
scriptingManager.reloadInterface(INTERFACEID) in every if statement